force-app/main/default/appMenus/AppSwitcher.appMenu-meta.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<AppMenu xmlns="http://soap.sforce.com/2006/04/metadata">
+    <appMenuItems>
+        <name>standard__Platform</name>
```

Comment:
@t-gul @bmtemakina please include your new app only. The other menu items are not about the task